content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -109,26 +109,26 @@ The upgrade workflow at high level is the following:
     [preflight] Running pre-flight checks.
     [upgrade] Making sure the cluster is healthy:
     [upgrade] Fetching available versions to upgrade to
-    [upgrade/versions] Cluster version: v1.15.2
-    [upgrade/versions] kubeadm version: v1.16.0
+    [upgrade/versions] Cluster version: v1.16.0
+    [upgrade/versions] kubeadm version: v1.17.0
 
     Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
     COMPONENT   CURRENT       AVAILABLE
-    Kubelet     1 x v1.15.2   v1.16.0
+    Kubelet     1 x v1.16.0   v1.17.0
 
     Upgrade to the latest version in the v1.16 series:
 
     COMPONENT            CURRENT   AVAILABLE
-    API Server           v1.15.2   v1.16.0
-    Controller Manager   v1.15.2   v1.16.0
-    Scheduler            v1.15.2   v1.16.0
-    Kube Proxy           v1.15.2   v1.16.0
-    CoreDNS              1.3.1     1.6.2
-    Etcd                 3.3.10    3.3.15
+    API Server           v1.16.0   v1.17.0
```

Comment:
nit: `v1.16.3` maybe?

Reply:
1.16.4 will be out soon.
these are just examples.